pkg/apis/scheduling/v1alpha1/labels.go

```diff
@@ -18,11 +18,11 @@ package v1alpha1
 
 // GroupNameAnnotationKey is the annotation key of Pod to identify
 // which PodGroup it belongs to.
-const GroupNameAnnotationKey = "scheduling.k8s.io/group-name"
+const GroupNameAnnotationKey = "scheduling.x-k8s.io/group-name"
```

Comment:
@k82cn as part of [this issue](https://github.com/kubernetes-sigs/kube-batch/issues/973) did we want to also update the annotation keys as such or was it just the group names? Just wanted to check/clarify

Reply:
let's update the annotation key accordingly to keep them align :)